styleguide/_themes/global/scss/global.scss

```diff
@@ -5,3 +5,4 @@
 @import '_fonts';
 @import '_forms';
 @import '_buttons';
+@import '_bootstrap_overrides';
```

Comment:
I think that may already be in the works.  Should we just wait till then or do it now?